views/js/qtiCreator/widgets/interactions/Widget.js

```diff
@@ -5,8 +5,9 @@ define([
     'taoQtiItem/qtiCreator/widgets/helpers/movable',
     'tpl!taoQtiItem/qtiCreator/tpl/toolbars/interaction',
     'tpl!taoQtiItem/qtiCreator/tpl/toolbars/okButton',
+    'taoQtiItem/qtiCreator/editor/gridEditor/content',
     '../../helper/classTitles'
-], function(_, $, Widget, movable, toolbarTpl, okButtonTpl, getQtiClassTitle){
+], function(_, $, Widget, movable, toolbarTpl, okButtonTpl, contentHelper, getQtiClassTitle){
```

Comment:
what is the risk of changing this signature ?

Reply:
It is not changing, it is just adding new import, see line 8